src/components/database/Queries.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import {database} from '../../firebaseFeatures.js';
+import type {UserType, PendingType, ActionType} from '../admin/FlowTypes.js';
+
+function sanitize(collection: string,
+    user: any): UserType | ActionType | PendingType {
+  if (collection === 'active-members') {
+    return ({
+      adminNote: user.adminNote,
+      email: user.email,
+      isAdmin: user.isAdmin,
+      joinDate: user.joinDate.toDate().toLocaleString(),
+      name: user.name,
+      needsAttention: user.needsAttention,
+      partnerEmail: user.partnerEmail,
+      count: user.count,
+      id: user.id,
+    }: UserType);
+  } else if (collection === 'pending-members') {
+    return ({
+      count: user.count,
+      email: user.email,
+      date: user.date.toDate().toLocaleString(),
+      partnerEmail: user.partnerEmail,
+    }: PendingType);
+  } else if (collection === 'actions') {
+    return ({
+      count: user.count,
+      date: user.date.toDate().toLocaleString(),
+      message: user.message,
+    }: ActionType);
+  } else {
+    throw Error('The collection you asked for doesn\'t exist.');
+  }
+}
+
+function dataSanitization(data: any,
```

Comment:
In general, I would recommend not to swap parameters between similar functions (i.e. `dataSanitization` and `sanitize`) around. This leads often to subtle errors.